Creating architecture diagram of dev environment.

kashan-01 opened the pull request:
Title: Dev Environment Architecture Diagram
Link: https://github.com/XgridInc/xc3/issues/69

    [PS-2]
- Added details of both 'dev' and 'prod' architecture diagram.

<br>

     [PS-1]
-  Added architecture diagram of dev environment.

Signed-off-by: Kashan Ali [<kashan.ali@xgrid.co>]

Commit:
Issue:
Title: Dev Environment Architecture Diagram
Link: #69

[PS-1]
- Modified architecture diagram(dev environment)

Signed-off-by: Kashan Ali <kashan.ali@xgrid.co>

yaseenzafar123 reviewed (approved):
LGTM

waji46 reviewed (approved):
LGTM

samanXgrid reviewed (approved):
LGTM

yaseenzafar123 reviewed (approved):
LGTM

faheem-khau9 commented on README.md:
```diff
@@ -27,9 +27,13 @@ Check the below video for a quick demo of XC3.
 
 - Provides cost optimization recommendation workflow without exposing your private information
```
Before this, let's talk about there are two different modes of deployment in XC3

kashan-01 replied:
Thanks for the feedback. You're right there two modes of deployment in XC3, So ?

kashan-01 replied:
@faheem-khau9 I've added details about 'dev' and 'prod' environment, please have a look. Thanks

Commit:
Title: Dev Environment Architecture Diagram
Link: https://github.com/XgridInc/xc3/issues/69

	[PS-2]
- Added details of both 'dev' and 'prod' architecture diagram

	[PS-1]
- Added architecture diagram of dev environment.

Signed-off-by: Kashan Ali [<kashan.ali@xgrid.co>]

faheem-khau9 reviewed (approved):
LGTM

muhdsaad reviewed (approved):
LGTM
